simpleauth/handler.py

```diff
@@ -291,7 +291,9 @@ def _oauth1_init(self, provider, auth_urls, extra=None):
 
     # make a request_token request
     client = self._oauth1_client(consumer_key=key, consumer_secret=secret)
-    resp, content = client.request(auth_urls['request'], "GET")
+    resp, content = client.request(auth_urls['request'], "POST",
+                                  body=urlencoded(
+                                      {'oauth_callback': callback_url}))
```

Comment:
I updated the indentation. If you meant the line with `urlencoded()`, it exceeds 80 character line thus I moved to a new line.